android/pytorch_android/style_transfer_cyclegan/README.md

```diff
@@ -0,0 +1,16 @@
+# DJL Android Demo
```

Comment:
I push your project under pytorch_android folder.

We also need add descriptions in pytorch_android/README.md